pgctl/cli.py

```diff
@@ -347,7 +346,7 @@ def services(self):
             for alias in self.pgconf['services']
             for service_name in self._expand_aliases(alias)
         ]
-        return uniq(services)
+        return tuple(sorted(set(services)))
```

Comment:
I believe I added this to de-flake one of the tests.
It's true that it could have better coverage.